Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -197,6 +205,27 @@ internal final class DiscoveryPageViewController: UITableViewController {
     }
   }
 
+  fileprivate func showShareSheet(_ controller: UIActivityViewController) {
+
+    controller.completionWithItemsHandler = { [weak self] activityType, completed, returnedItems, error in
+
+      self?.shareViewModel.inputs.shareActivityCompletion(
+        with: .init(activityType: activityType,
+                    completed: completed,
+                    returnedItems: returnedItems,
+                    activityError: error)
+      )
+    }
+
+//    if UIDevice.current.userInterfaceIdiom == .pad {
+//      controller.modalPresentationStyle = .popover
+//      let popover = controller.popoverPresentationController
+//      popover?.sourceView = self.shareButton
+//    }
```

Comment:
@cdolm92 have you tested on ipad? this needs to be presented as a popover or it will error.